src-docs/src/assets/version-selector.js

```diff
@@ -0,0 +1,230 @@
+
+const PREFIX = 'v';
+
+const tpl = `
+    <style>
+    :host {
+        display: inline-block;
+        position: relative;
+        box-sizing: border-box;
+        font-size: 12px;
+        user-select: none;
+        margin: 3px;
+        text-align: left;
+    }
+    
+    * {
+        box-sizing: border-box;
+    }
+    
+    #root {
+        text-decoration: none;
+        color: #FFFFFF;
+        // background-color: #00509c;
+        background-color: rgb(211, 218, 230);
+        color: rgb(0, 0, 0);
+        background-image: var(--normal-bg);
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.25), 0 4px 10px rgba(0, 0, 0, 0.12);
+        border-radius: 4px;
+        padding: 0.3em 3em 0.3em 1em;
+        margin: 0;
+        
+        position: relative;
+        display: block;
+        z-index: 2;
+        cursor: pointer;
+    }
+    
+    #root:hover {
+        background-image: var(--hover-bg);
+    }
+    
+    #root:focus {
+        outline: none;
+    }
+    
+    :host(:not([aria-expanded="true"])) #root:focus,
+    #root:focus:hover {
+        box-shadow: 0 0 0 3px rgba(0, 0, 255, 0.25);
+    }
+    
+    #root:before {
+        content: "";
+        position: absolute;
+        top: 5px;
+        bottom: 5px;
+        width: 0;
+        border-width: 0 1px;
+        border-color: #000 rgba(0, 0, 0, .2) #000 rgba(255, 255, 255, .6);
+        right: 2em;
+        border-style: solid;
+        background-blend-mode: multiply;
+    }
+    
+    #root > svg {
+        position: absolute;
+        right: .5em;
+        top: .6em;
+    }
+    
+    #dropdown {
+        position: absolute;
+        min-width: calc(100% - 2px);
+        top: 100%;
+        left: 0;
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.25), 0 4px 10px rgba(0, 0, 0, 0.12);
+        
+        margin: -5px 1px 0 1px;
+        padding-top: 5px;
+        white-space: nowrap;
+        border-radius: 0 0 4px 4px;
+        
+        background: #fff;
+        z-index: 1;
+    }
+    
+    :host(:not([aria-expanded="true"])) #dropdown {
+        display: none;
+    }
+    
+    #spacer {
+        appearance: none;
+        visibility: hidden;
+        pointer-events: none;
+        height: 0;
+        margin: 0 1px;
+        overflow: hidden;
+    }
+    
+    #spacer > a,
+    #dropdown > a {
+        display: block;
+        white-space: nowrap;
+        padding: 0.3em calc(3em - 1px) 0.3em calc(1em - 1px);
+        border-bottom: 1px solid #eee;
+        text-decoration: none;
+        color: var(--link-color);
+        position: relative;
+    }
+    
+    #dropdown > a:last-child {
+        border: 0;
+    }
+    
+    #dropdown > a:hover {
+        background: #efefef;
+    }
+    
+    a.latest:after {
+        content: "LATEST";
+        position: absolute;
+        right: .4rem;
+        font-size: 0.6em;
+        font-weight: 700;
+        top: 50%;
+        transform: translateY(-50%);
+        color: #999;
+    }
+    </style>
```

Comment:
Why are we not adding separate css file instead of adding here?

Reply:
its a web component, not active part of OUI. so not creating un-necessary css